### PR TITLE
16.0

### PR DESCRIPTION
**Pull request by benoit-bertholon**
update v16 for auth_api_key

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Please follow the git format-patch method to preserve the commit history and check all the other instructions from https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch.

File: auth_api_key/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Auth Api Key",
+    "summary": """
+        Authenticate http requests from an API key""",
+    "version": "16.0.1.1.1",
```
Comment (StefanRijnhart) [inline auth_api_key/__manifest__.py:8]: Reset to 16.0.1.0.0

File: auth_api_key/models/auth_api_key.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @api.model
+    @tools.ormcache("key")
+    def _retrieve_uid_from_api_key(self, key):
+        return self._retrieve_api_key(key).user_id.id
+
+    def _clear_key_cache(self):
+        self._retrieve_api_key_id.clear_cache(self.env[self._name])
+        self._retrieve_uid_from_api_key.clear_cache(self.env[self._name])
+
+    @api.model
```
Comment (StefanRijnhart) [inline auth_api_key/models/auth_api_key.py]: Refactor as `model_create_multi` to prevent Odoo from emitting a warning.

**Comment by benoit-bertholon** [comment]
new pr following guidelines

**Comment by StefanRijnhart** [comment]
https://github.com/OCA/server-auth/pull/451

**Pull request closed without merging**